CHANGELOG.md

```diff
@@ -5,6 +5,300 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 8.0.0-beta.0
+
+_Feb 27, 2025_
+
+We'd like to offer a big thanks to the 18 contributors who made this release possible. Here are some highlights ✨:
+
+- 🔧 End of alpha and all its breaking changes 🎉
+- 📊 Decouple `margin` and `axis-size`. A new API that enable multiple axis (#16418) @JCQuintas
+- 🗺️ Added Bangla (bn-BD) locale
+- 🗺️ Improve Russian (ru-RU) and Hungarian (hu-HU) locale on the Data Grid
+
+Special thanks go out to the community contributors who have helped make this release possible:
```

Comment:
This feels redundant. I'll update the action later, but could you please change that?

```suggestion
Special thanks go out to the community members for their contributions:
```

Reply:
https://github.com/mui/mui-x/pull/16751 ^^